### PR TITLE
GCW-2073 Fixed issue : cannot save description in accept tab

### PR DESCRIPTION
**Pull request by abulsayyad123**
Hi Team,
This is PR for Jira ticket GCW-2073-Cannot save description in the accept tab
https://jira.crossroads.org.hk/browse/GCW-2073.
I have added fix for the 2 scenarios discussed in the ticket and added test cases for it.
Review the ticket and let me know if any change is required.

Thanks.

### CODE REVIEW

File: app/models/package.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ export default DS.Model.extend({
     return this.get('packageType.name');
   }),
 
+  changedNotes: Ember.computed('notes', function(){
```
Comment (patrixr) [inline app/models/package.js]: I noticed there was an Ember.computed.alias method, would that be appropriate to this case ?

Comment (namrataukirde) [reply app/models/package.js]: Yes, I agree with you @patrixr, It will work according to me.

File: app/components/package-select-list.js [hunk 2]
```diff
@@ -12,8 +12,11 @@ export default SelectList.extend( {
   didRender() {
     this._super(...arguments);
     if (!this.get('didUpdatedOnce') && this.get('pkg') !== null && this.get('pkg.item.packageType') !== null){
-        var packageType = this.get('pkg.packageType') || this.get('pkg.item.packageType');
-        this.set('pkg.notes', packageType.get('name'));
+        var packageType = this.get('pkg.item.packageType');
+
+        if (this.get('pkg.notes') === packageType.get('name') || this.get('pkg.notes') === "" || (this.get('pkg.packageType.name') !== packageType.get('name'))) {
```
Comment (Darshan-Chauhan) [inline app/components/package-select-list.js]: @abulsayyad123  Would it be better to cache **pkg** and extract out the conditions in a function returning only true or false??

File: app/models/package.js [hunk 3]
```diff
@@ -48,6 +48,8 @@ export default DS.Model.extend({
     return this.get('packageType.name');
   }),
 
+  changedNotes: Ember.computed.alias("notes"),
```
Comment (Darshan-Chauhan) [inline app/models/package.js]: @abulsayyad123 , it doesn't make any sense to me to alias a model attribute in model itself. 
What's the issue with using `model.notes`?

**Comment by Darshan-Chauhan** [comment]
@abulsayyad123 , good work. Good to see the code-coverage change.
Added some comments.

File: app/models/package.js (hunk 3 above)
Comment (abulsayyad123) [reply app/models/package.js]: It is giving error related to binding which says that you cannot edit binded variable i.e notes. That is why i had alias notes variable.

**Pull request merged**